src/main/webapp/app/entities/patient/patient-details.component.ts

```diff
@@ -21,6 +35,16 @@ export default defineComponent({
 
     const previousState = () => router.go(-1);
     const patient: Ref<IPatient> = ref({});
+    const poidsPatient: Ref<Array<Object>> = ref([]);
+    const EPAPatient: Ref<Array<Object>> = ref([]);
+    const patientIMC: Ref<Number> = ref(0);
+    const weightChartData: Ref<Object> = ref({});
+    const EPAChartData: Ref<Object> = ref({});
+    const chartOptions: Ref<Object> = ref({
+      responsive: true,
+    });
+    const weightChartLoaded: Ref<Boolean> = ref(false);
+    const EPAChartLoaded: Ref<Boolean> = ref(false);
 
     const retrievePatient = async patientId => {
```

Comment:
retrievePatient implicitly has type any because it does not have a type annotation